plugin/setup.py

```diff
@@ -78,7 +78,8 @@ def run(self):
 class Test(setuptools.command.install.install):
 
     def run(self):
-        self.spawn(['ant', 'unit-tests-only'])
+        # self.spawn(['ant', 'unit-tests-only'])
```

Comment:
add a TODO: reactivate tests

Reply:
This and other TODOs moved to https://trello.com/c/vyZSm0fj/15-openlayers-5-build-cleanup